linera-service/tests/end_to_end_tests.rs

```diff
@@ -681,14 +681,23 @@ async fn test_end_to_end_faucet(config: impl LineraNetConfig) {
     let chain2 = outcome.chain_id;
     let message_id = outcome.message_id;
 
+    // Use the faucet directly to initialize client 3.
+    let client3 = net.make_client();
+    let outcome = client3.wallet_init(&[], Some(&faucet.url())).await.unwrap();
+    let chain3 = outcome.unwrap().chain_id;
+    assert_eq!(
+        chain3,
+        client3.get_wallet().unwrap().default_chain().unwrap()
+    );
+
     faucet.ensure_is_running().unwrap();
     faucet.terminate().await.unwrap();
 
-    // Chain 1 should have paid two tokens.
+    // Chain 1 should have paid four tokens.
```

Comment:
Is 'paid' a little misleading here? Maybe something like 'has had 4 tokens redeemed from faucet'.

Reply:
I clarified the comment a bit.